lib/form/paragraphtypes.flow

```diff
@@ -66,18 +66,31 @@ export {
 		// A space, newline or text element wrapped with an index of where it came from,
 		// as well as the coordinates and line number of where it is
 		InspectElement(
+			inspector : ParaElementInspector,
+			element : InspectableElement,
+		);
+			InspectableElement ::= Space, Form, LinePart;
+
+		AllowLineBreak = TextFragments(Cons(Space(Empty()), EmptyList()));
+
+		GeneralTextFragments : (elements : List<GeneralWrapElement>, style : [CharacterStyle]);
+			GeneralWrapElement ::= GeneralText, GeneralSpace, NewLine, GeneralLinePart, GeneralInspectElement, EmptyLineElement;
+				GeneralText(txt : string);
+				GeneralSpace(str : string);
```

Comment:
@artxp , check it please, you told us that Space(form) is used in some asia languages related scenarios

Reply:
I took it into account resolving conflicts with @artxp 's commit